### PR TITLE
UI/UX issue

### LINKED ISSUE #5: User Interface/User Experience (UI/UX)
| **Task** | Use design tools and concepts to build a wireframe and mockup using UI best practices and laws of UX |
| - | - |
| **Conditions** | Member has a working Figjam/Figma account |
| **Standards** | <ul><li>Use markdown to write copy for a webpage</li><li>Build a wireframe in Figjam and insert the copy</li><li>Create a design mock up in Figma implementing the wireframe using UI Design Best Practices</li><li>Use at least five different UX laws in your design and call out which ones you used in your markup file and the reasons for using them</li><li>Add links to the wireframe and mock up to the Markdown and submit a PR</li></ul> |

## References
- [User Interface Best Practices](https://www.usability.gov/what-and-why/user-interface-design.html)
- [Laws of UX](https://lawsofux.com/)
- [UI/UX Design Tutorial](https://www.youtube.com/watch?v=c9Wg6Cb_YlU)
- [Figjam 101](https://help.figma.com/hc/en-us/articles/1500004362321-Guide-to-FigJam)
- [Figma 101](https://designlab.com/figma-101-course/introduction-to-figma/#:~:text=Getting%20started%20in%20Figma%20is,and%20we'll%20get%20started!)

### PR DESCRIPTION
**Pull request by JVSONV**
created README file resolves #5

### CODE REVIEW

**Comment by axecopfire** [comment]
There is ALOT of copy in your design for one page. If we are looking for conversion, your users are going to look at that wall of text and bounce. What do you actually want your users to look at and do. Fitt's Law, time to acquire a target, what's your target? Hick's Law, you're showing all three text blobs at once, what do you want them to do with that information. Law of Proximity, objects that are near tend to be grouped together. All these text blobs are grouped together, making me think you want me to read it all.

File: issueWork/UI/UX__issue/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# UI / UX issue  
+
+## Changes made:  
+
+> 1. Complete overhall of webpage layout based on previously received feedback. Essentially creating a clean and somewhat minimal style.
+ - ### Laws:  
+   - **Aesthetic-Usability**
+   - **Jakob's Law**
```
Comment (axecopfire) [inline issueWork/UI/UX__issue/README.md:8]: If this site is supposed to look like other sites, tell me what other site this looks like.

File: issueWork/UI/UX__issue/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+   - **Hick's Law**
```
Comment (axecopfire) [inline issueWork/UI/UX__issue/README.md:9]: What decision have you optimized with this new design. How did you optimize it?

File: issueWork/UI/UX__issue/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+> 2. Attached and created an accordin to the hero section's breif about me. Which when clicked expands to a larger more indepth about me section.
```
Comment (axecopfire) [inline issueWork/UI/UX__issue/README.md:10]: Use [hemingway](https://hemingwayapp.com/) or Grammarly when writing copy. You have a lot of spelling errors.

**Comment by JVSONV** [comment]
> 

The object was to have the big about me hidden by default so the viewer could decide what they wanted to see or not see. I thought the point of the copy write document was meant to be transferred to here. If we were supposed to trim it down then I will.

File: issueWork/UI/UX__issue/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+   - **Law of Common Region**
+
+> 3. Created a Projects section. Currently grouping three mock projects with title, description, and what poblem it solved. May introduce languages and technology used to create project. This is dependant on feedback received.  
+ -  ### Laws:    
+    - **Law of Similarity**
+    - **Tesler's Law**
+
+> 4. Though not completed the contact form with be a modal with only 3 inputs. 
+ - ### Law:  
+   - **Postel's Law**
```
Comment (axecopfire) [inline issueWork/UI/UX__issue/README.md:23]: Interesting, how does Postel's law apply here? This is about handling variable input

File: issueWork/UI/UX__issue/README.md (hunk 1 above)
Comment (JVSONV) [reply issueWork/UI/UX__issue/README.md:8]: This is a universal reference as most simple developer portfolios I have seen looked similar.

File: issueWork/UI/UX__issue/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+> 5.  In the final implementation the nav will be affixed to the top of the page. Idea here is to utilize Miller's Law in a way that the viewer is only seeing the nav and three projects. This way upon review of projects the contact me button is not out of sight.
+- ### Law:  
+   - **Miller's Law**
```
Comment (axecopfire) [inline issueWork/UI/UX__issue/README.md:27]: I kinda see where you're going with Miller's Law here. I don't know if it completely fits. Though I do see that you're trying to keep that nav simple, which I do think is good design.

File: issueWork/UI/UX__issue/README.md (hunk 2 above)
Comment (JVSONV) [reply issueWork/UI/UX__issue/README.md:9]: My main point here with Hick's Law was but removing any unneeded fluff from the portfolio. Giving the hiring manager what they want to see as streamlined as possible. By not hiding the project section behind a lengthy road to get there. They will have more time with my projects which are the real "prof in the pudding" items. And if they chose they can read more about me if I meet their standards to do so through my projects.

File: issueWork/UI/UX__issue/README.md (hunk 4 above)
Comment (JVSONV) [reply issueWork/UI/UX__issue/README.md:23]: Postel's Law here would mainly be:
- Anticipate virtually anything in terms of input, access, and capability while providing a reliable and accessible interface.
- The more we can anticipate and plan for in design, the more resilient the design will be.
Accessibility will be the top importance here as who do know who will be reviewing your portfolio. 
Secondly the modal is in anticipation of when one might want to get in contact with me. But making it a modal I allow the viewer to contact me the second they wish too. All while not pulling them away from the content that caused them to do so. (ie: not scroll jacking them to a different section on the page)

File: issueWork/UI/UX__issue/README.md (hunk 5 above)
Comment (JVSONV) [reply issueWork/UI/UX__issue/README.md:27]: Right and with the current way its setup I tend to thing the about and project links are not needed and may remove them.

File: issueWork/UI/UX__issue/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# UI / UX issue  
+
+## Changes made:  
+
+> 1. Complete overhall of webpage layout based on previously received feedback. Essentially creating a clean and somewhat minimal style.
+ - ### Laws:  
+   - **Aesthetic-Usability**
```
[the remark this answers is not in the document]
Comment (axecopfire) [reply issueWork/UI/UX__issue/README.md]: Please add this response to the markdown file.

File: issueWork/UI/UX__issue/README.md (hunk 1 above)
Comment (axecopfire) [reply issueWork/UI/UX__issue/README.md]: Can you add specific references for simple developer portfolios that you like here?

File: issueWork/UI/UX__issue/README.md (hunk 2 above)
Comment (axecopfire) [reply issueWork/UI/UX__issue/README.md]: I don't know what the UX law is. But if you show someone a button, they're gonna click it. I bet you 8 times out of 10 folks are gonna click the learn more button thinking that's where you're telling them to go. I doubt a user is gonna see that scroll, to projects then scroll back to the about. I mean maybe, but you're making a bet that your button isn't enticing, which there may be a better way to showcase your projects if that's what you want to do.

File: issueWork/UI/UX__issue/README.md (hunk 3 above)
Comment (axecopfire) [reply issueWork/UI/UX__issue/README.md]: No worries, documentation is a huge skill that folks are hiring for. So working on communication is going to be key. It looks a lot better after your edits. Thanks.

**Review by axecopfire — APPROVED** [review]

**Pull request merged**